build/8-to-9-migration/migrate.sh

```diff
@@ -6,14 +6,15 @@ mvn_cmd="mvn -B ${BUILD_MVN_OPTS:-}"
 optaplanner_file="$script_dir_path/optaplanner-quarkus3.yaml"
 optaplanner_root="$script_dir_path/../.."
 
+# Install artifacts locally.
+${mvn_cmd} clean install -Dquickly
```

Comment:
The 8.x local build has to run before we remove custom Hibernate score types; otherwise, the dependency plugin complains about an unused dependency on `hibernate-core`, which is removed by the rewrite plugin later in this script.